samples/bring-your-own-code/react/bank-loan-application/powerpages.config.json

```diff
@@ -0,0 +1,6 @@
+{
+    "$schema": "https://json.schemastore.org/powerpages.config.json",
+    "compiledPath": "dist",
+    "defaultLandingPage": "index.html",
+    "siteName": "Bank Loan Application"
```

Comment:
Please add `includeSource: true` as well and add check in `source-files` metadata folder.